### ISSUE TITLE
`min-width` is not a valid property

### ISSUE THREAD

**Issue by lehins** [cdepillabout/termonad#21]
Wanted to give `termonad` a try, but out of the box setup failed. Followed installation instructions on Ubuntu 16.04:

```
$ apt-get install gobject-introspection libgirepository1.0-dev libgtk-3-dev libvte-2.91-dev
$ git clone https://github.com/cdepillabout/termonad
$ cd termonad/
$ stack install
...
Copied executables to /home/user/.local/bin:
- termonad
$ termonad

termonad: <data>:3:11'min-width' is not a valid property name (3)
```

**Comment by cdepillabout** [comment]
@lehins Thanks for this bug report.  I appreciate that you took the time to file this.  I've tried to fix this in #22.  Could you try using that branch and see if it fixes your problems?

**Comment by lehins** [comment]
@cdepillabout The fix worked. Thank you. :)

**Comment by cdepillabout** [comment]
Okay thanks, I merged in #22, so now the `master` branch should be usable by you and anyone else on Ubuntu 16.04.
